OBAKit/UI/OBATheme.h

```diff
@@ -23,6 +28,11 @@
  */
 + (void)setAppearanceProxies;
 
+/**
+ Create custom navigation bar title view.
+ */
++ (UIView *)createNavbarTitleViewWithTitle:(NSString *)title subtitle:(NSString *)subtitle andStyle:(OBAAppearanceNavBarTitleViewStyle)style;
```

Comment:
I think this is moot given some later feedback, but just a good thing to keep in mind :)